build_posix/configure.ac.in

```diff
@@ -155,7 +155,9 @@ if test "$wt_cv_enable_java" = "yes"; then
 fi
 
 if test "$wt_cv_enable_python" = "yes"; then
-	AM_PATH_PYTHON([2.6])
+	# If we have Python 2.x, use python3
+	CHECK_MAJOR_VERSION([PYTHON_VERSION], [2], PYTHON=python3, :)
```

Comment:
Like the smiling face here :)  Is the effect of `:` a no-op?
And what's the reasoning of using Python3 if we have 2.x? What if we only have 3.x?

Reply:
Yes, `:` is a no-op in the shell.  This macro argument is an action to do if the condition is false.  It has to be something other than blank, blank will give a shell syntax error.
That all said, I think my reasoning in using this macro was incorrect, it's not needed, so I removed it.